target/linux/ath79/dts/qca9558_domywifi_dw33d.dts

```diff
@@ -0,0 +1,215 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9557.dtsi"
+
+/ {
+	compatible = "domywifi,dw33d", "qca,qca9558";
+	model = "DomyWifi DW33D";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &status;
+		led-failsafe = &status;
+		led-running = &status;
+		led-upgrade = &status;
+	};
+
+	gpio_leds: leds {
+		compatible = "gpio-leds";
+
+		status: status {
+			label = "dw33d:blue:status";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		usb0 {
+			label = "dw33d:blue:usb0";
+			gpios = <&gpio 4 GPIO_ACTIVE_LOW>;
+			trigger-sources = <&hub_port0>;
+		};
```

Comment:
So, why did you change from "mmc" to "usb0"?
(This will need migration for labels, too.)

Reply:
This device does not have an mmc flasher, but a usb card reader. I hope to avoid ambiguity.

Reply:
If the usb card reader is already on board and this led is for mmc activity I think it should be named mmc instead.

Reply:
Ok, continue to use the mmc name.

Reply:
But then it shouldn't trigger for the usb either?

Reply:
The card reader is embedded in the circuit board and occupies usb0.Panel led display properly.